dev-lang/rust-bin/rust-bin-1.71.1.ebuild

```diff
@@ -62,6 +62,10 @@ pkg_pretend() {
 	if [[ "$(tc-is-softfloat)" != "no" ]] && [[ ${CHOST} == armv7* ]]; then
 		die "${CHOST} is not supported by upstream Rust. You must use a hard float version."
 	fi
+
+	if has_version sys-devel/clang-runtime[libcxx]; then
```

Comment:
add a bug ref above it then lgtm

Reply:
(we may want to mask rust-bin on the llvm profiles but let's see if anyone complains from this first and/or gives us a workaround)